src/command/dependencies/filter.rs

```diff
@@ -2,25 +2,29 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-use std::collections::{HashMap, HashSet};
-
+use std::collections::HashMap;
+use std::collections::HashSet;
+
+use petgraph::Direction;
+use petgraph::graph::NodeIndex;
+use petgraph::stable_graph::EdgeIndex;
+use petgraph::visit::Bfs;
+use petgraph::visit::EdgeRef;
+use petgraph::visit::IntoEdgeReferences;
 use ra_ap_hir::{self as hir};
 use ra_ap_ide::{self as ide};
 use ra_ap_syntax::ast;
 
-use petgraph::{
-    Direction,
-    graph::NodeIndex,
-    stable_graph::EdgeIndex,
-    visit::{Bfs, EdgeRef, IntoEdgeReferences},
-};
-
-use crate::{
-    analyzer::{self, has_test_cfg, is_test_function},
-    graph::{Edge, Graph, GraphWalker, Node, Relationship},
-};
-
 use super::options::Options;
+use crate::analyzer::has_test_cfg;
+use crate::analyzer::is_test_function;
+use crate::analyzer::{self};
+use crate::graph::Edge;
+use crate::graph::Graph;
+use crate::graph::GraphWalker;
+use crate::graph::Node;
+use crate::graph::Relationship;
+use crate::item::ItemVisibility;
```

Comment:
Woopsie, my editor reordered imports.

Sorry about that. I'll fix that in a minute!